app/views/quotes/_quote.html.erb

```diff
@@ -0,0 +1 @@
+<%= quote.author %> said, "<%= quote.body %>", in the <%= quote.location %>
```

Comment:
Suggestion for the future: html has has `<q></q>` tag for inline quotes, as well as `<blockquote></blockquote>` for (as the name implies) block quotes.